### PR TITLE
[SFT-627] - Remove the Stats widget

### PR DESCRIPTION
**Pull request by seandelaney**

### CODE REVIEW

**Review by gustavs-gutmanis — APPROVED** [review]
Looks great 👌🏻
Lets wait for @kjmartens approval as well

**Review by kjmartens — APPROVED** [review]
Looks great! Appreciate the migration as well.

**Pull request merged**
